vite.config.js

```
@@ -2,6 +2,8 @@ import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react';
 import { VitePWA } from 'vite-plugin-pwa';
 import svgr from 'vite-plugin-svgr';
+import fs from 'fs';
+import path from 'path';
```

ESLint

[error] 'path' is defined but never used. Allowed unused vars must match /^[A-Z_]/u.

(no-unused-vars)